integration_tests/array_expr_02.py

```diff
@@ -0,0 +1,37 @@
+from ltypes import i32, f32, TypeVar
+from numpy import empty, sqrt
+
+n: i32
+n = TypeVar("n")
+
+def modify(array_a: f32[:], n: i32) -> f32[n]:
+    return sqrt(array_a)
+
+def verify(array_a: f32[:], array_b: f32[:], result: f32[:], size: i32):
+    i: i32
+    eps: f32
+    eps = 1e-6
+
+    for i in range(size):
+        assert abs(array_a[i]**2 + sqrt(array_b[i]) - result[i]) <= eps
+
+def f():
+    i: i32
+    j: i32
+    k: i32
+
+    array_a: f32[256] = empty(256)
+    array_b: f32[256] = empty(256)
+    array_c: f32[256] = empty(256)
```

Comment:
Here we need this:
```suggestion
    array_a: f32[256] = empty(256, dtype=float32)
    array_b: f32[256] = empty(256, dtype=float32)
    array_c: f32[256] = empty(256, dtype=float32)
```
Otherwise CPython would use a different type than LPython.